server/prisma/prisma.yml

```diff
@@ -10,6 +10,7 @@ generate:
     output: ../src/generated/prisma-client/
 
 # Ensures Prisma client is re-generated after a datamodel change.
+# TODO: Why is this failing when running "prisma deploy"
```

Comment:
There were issue with Prisma demo servers last night, we won't be using them longterm. Id recommend setting up a local cluster for development. We can chat about it in our Wednesday meeting.

Reply:
👍 great sounds good!